pyproject.toml

```diff
@@ -14,7 +14,7 @@ urdfpy = "^0.0.22"
 Sphinx = { version = "4.2.0", optional = true }
 sphinx-rtd-theme = { version = "1.0.0", optional = true }
 sphinxcontrib-napoleon = { version = "0.7", optional = true }
-motion-planning-scenes = {version = "^0.1.19", optional = true}
+motion-planning-scenes = {git="https://github.com/GijsGroote/motion_planning_scenes.git",  rev='gijs_develop'}
```

Comment:
We must merge the differences between your develop branch and the current version before we can merge this.

Reply:
@GijsGroote Is this change really needed? Otherwise, we could merge this PR without having to wait for the developments on motion_planning_scenes.